cluster-autoscaler/core/utils.go

```diff
@@ -491,3 +491,13 @@ func UpdateClusterStateMetrics(csr *clusterstate.ClusterStateRegistry) {
 	readiness := csr.GetClusterReadiness()
 	metrics.UpdateNodesCount(readiness.Ready, readiness.Unready+readiness.LongNotStarted, readiness.NotStarted)
 }
+
+func getOldestCreateTime(pods []*apiv1.Pod) time.Time {
+	oldest := time.Now()
```

Comment:
I think in general we should use currentTime parameter of RunOnce, not time.Now(). Those timestamps may be different since we're already after the FilterOutSchedulable call, which can take noticeable amount of time. But we're still looking at cluster state as of 'currentTime' parameter.

Reply:
oldest is used only for safety, if the list of pods is empty, this value is unlikely to be used.

Reply:
Right, I misunderstood how we use the timestamp. You're correct, it doesn't matter.